apps/api/src/bootstrap.ts

```diff
@@ -39,9 +39,11 @@ if (process.env.SENTRY_DSN) {
 
 // Validate the ENV variables after launching SENTRY, so missing variables will report to sentry
 validateEnv();
+const otelSDK = getOTELSDK(packageJson.name);
 
 export async function bootstrap(expressApp?): Promise<INestApplication> {
   BullMqService.haveProInstalled();
+  await otelSDK.start();
```

Comment:
Is it possible to only enable OTEL if an env variable indicating it was provided?